Directory.Build.targets

```diff
@@ -12,6 +12,7 @@
   <Import Project="$(RepositoryEngineeringDir)generators.targets" />
   <Import Project="$(RepositoryEngineeringDir)python.targets" />
   <Import Project="$(RepositoryEngineeringDir)resolveContract.targets" Condition="'$(IsSourceProject)' == 'true'" />
+  <Import Project="$(RepositoryEngineeringDir)intellisense.targets" Condition="'$(MSBuildProjectExtension)' != '.pkgproj'" />
```

Comment:
Instead of adding the intellisense Targets in packaging.targets, it feels cleaner to have them defined in a separate file. Since some of the elements defined in this new file are required by packaging.targets, the Import is added before packaging targets. The condition seems to make sense here too.